src/stores/extensionStore.ts

```diff
@@ -120,14 +92,6 @@ export const useExtensionStore = defineStore('extension', () => {
     return extensions.value.some((ext) => !isCoreExtension(ext.name))
   })
 
-  // Some core extensions are registered before the store is initialized, e.g.
```

Comment:
Ref: https://github.com/Comfy-Org/ComfyUI_frontend/pull/2047

There is no longer extension registered before store is initialized.